datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/KafkaBasedConnectorTaskMetrics.java

```diff
@@ -52,7 +56,11 @@
   private final AtomicLong _numAutoPausedPartitionsAwaitingDestTopic = new AtomicLong(0);
   private final AtomicLong _numTopics = new AtomicLong(0);
 
-  KafkaBasedConnectorTaskMetrics(String className, String metricsKey, Logger errorLogger) {
+  private final boolean _enablePollDurationMillisMetric;
+  private final Histogram _pollDurationMs;
```

Comment:
nit: please change the name from _pollDurationMs to _pollDurationMsMetric. the name is confusing.

Reply:
done